airflow/api_connexion/endpoints/task_instance_endpoint.py

```diff
@@ -43,7 +43,7 @@
 
 @security.requires_access(
     [
-        ("can_read", "Dag"),
+        ("can_read", "AllDags"),
```

Comment:
Why is this one AllDags, not Resource.DAGS?

Reply:
Ah this is because this endpoint is new. Updated.